server/apps/accounts/functions.py

```
@@ -0,0 +1,45 @@
+)
+def sync_user(ctx: inngest.ContextSync) -> str:
+    event = ctx.event
+    data = event.data
+    clerk_id = data.get("id")
+
+    match event.name:
+        case (
+            ClerkWebhookEvent.USER_CREATED.value | ClerkWebhookEvent.USER_UPDATED.value
+        ):
+            email = (data.get("email_addresses") or [{}])[0].get("email_address")
+            user, created = User.objects.update_or_create(
```

Ruff (0.16.1)

[warning] Unpacked variable `user` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)